openbb_terminal/cryptocurrency/discovery/dappradar_model.py

```diff
@@ -64,7 +65,8 @@ def _make_request(url: str, verbose: bool = True) -> Optional[dict]:
     response = requests.get(url, headers=headers)
     if not 200 <= response.status_code < 300:
         if verbose:
-            console.print(f"[red]dappradar api exception: {response.text}[/red]")
+            console.print(
```

Comment:
formatting wrong. did you configure precommit?

Reply:
My VS Code formatting settings weren't configured correctly, but sorted it already.